project/plugins.sbt

```diff
@@ -1,6 +1,11 @@
 // these comment markers are for including code into the docs
 //#sbt-multi-jvm
-addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.4.0")
+//addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.4.0")
+libraryDependencies += Defaults.sbtPluginExtra(
+  "com.eed3si9n" % "sbt-assembly" % "0.14.5",
+  (sbtBinaryVersion in pluginCrossBuild).value,
+  (scalaBinaryVersion in pluginCrossBuild).value
+)
 //#sbt-multi-jvm
```

Comment:
note that this is inside a doc snippet, is this what we want to show in docs?

Reply:
That's just for testing it right inside the akka build.